spec.html

```diff
@@ -3901,7 +3901,7 @@ <h1>Well-Known Intrinsic Objects</h1>
     <h1>ECMAScript Specification Types</h1>
     <p>A specification type corresponds to meta-values that are used within algorithms to describe the semantics of ECMAScript language constructs and ECMAScript language types. The specification types include Reference, List, Completion Record, Property Descriptor, Environment Record, Abstract Closure, and Data Block. Specification type values are specification artefacts that do not necessarily correspond to any specific entity within an ECMAScript implementation. Specification type values may be used to describe intermediate results of ECMAScript expression evaluation but such values cannot be stored as properties of objects or values of ECMAScript language variables.</p>
 
-    <emu-clause id="sec-list-and-record-specification-type">
+    <emu-clause id="sec-list-and-record-specification-type"
```

Comment:
```suggestion
    <emu-clause id="sec-list-and-record-specification-type">
```

Reply:
Yeah that's already been fixed. See https://github.com/tc39/ecma262/pull/2848/commits/81129217591513ed4c4b6dbd0274e356249bafa0.